examples/Viewer.tsx

```diff
@@ -134,6 +136,16 @@ class Viewer extends React.Component<{}, ViewerState> {
         e.preventDefault();
     };
 
+    public onError = (error: FrontEndError) => {
+        if (error.level === ErrorLevel.ERROR) {
+            window.alert(`ALERT, something is broken: ${error.htmlData}`);
+        } else if (error.level === ErrorLevel.WARNING) {
+            window.alert(`User warning, but not terrible: ${error.htmlData}`);
+        } else {
+            console.log(`Just for your info: ${error.htmlData}`);
```

Comment:
here's the case I brought up where you are `console.log`ing html and it would make sense to have a non-html message

Reply:
i wonder if there is a utility function to just remove any `<tags>` from a string

Reply:
... or maybe the console.log should log error.message?

Reply:
I was a bit mistaken when we talked this morning: the errors always have a message, they may also have htmlData. I updated the example viewer to log just the message